projects/admin/src/app/record/detail-view/document-detail-view/holding-item/holding-item.component.ts

```diff
@@ -49,10 +51,18 @@ export class HoldingItemComponent {
    */
   constructor(
     private recordUiService: RecordUiService,
-    private recordPermissionMessage: RecordPermissionMessageService,
-    private userService: UserService
+    private userService: UserService,
```

Comment:
_recordUiService and _ userService?
Maybe check other private ones.

Reply:
done